### ISSUE TITLE
[Meteor 1.3 rc.3] webpack build too long.

### ISSUE THREAD

**Issue by lyquocnam** [thereactivestack/meteor-webpack#113]
i just create new project with meteor 1.3 rc.3 and try to build with webpack.
it take long time about 3 to 5 minutes. awesome.
i don't know what problem. about meteor 1.3rc3 or webpack.
someone testing ?

**Comment by lyquocnam** [comment]
i checked meteor 1.3 build with emascript, all ok ! but with webpack it goes to long, sometime hang at:

```
=> Started proxy.
=> Started MongoDB.
webpack built 30524b13e1d0ee294b64 in 84675ms/
   Building the application                  -

```

**Comment by lyquocnam** [comment]
Meteor 1.3 rc.4 too

**Comment by swennemans** [comment]
@lyquocnam is this only with 1.3 RC? How is your experience with 1.2?

**Comment by lyquocnam** [comment]
not try with 1.2 yet mr @swennemans i think it ok, because this fine on 1.3 beta.12

**Comment by lyquocnam** [comment]
@swennemans this package run well on meteor 1.2 [kickstart here](https://github.com/thereactivestack/kickstart). all features like meteor 1.3 awesome.

**Comment by D1no** [comment]
Yes, the build is incredibly slow on rc.6 (!) from 10 sec to 10 min

**Comment by lyquocnam** [comment]
omg mr @eXon we miss you so much :100:  many meteor version release too :3

**Comment by D1no** [comment]
the rc.8 is better

**Comment by lyquocnam** [comment]
ya, this may resolved build time but i have this error #120 what about you @D1no ?
